src/Entities/Schema.php

```diff
@@ -165,6 +165,9 @@ public function __construct()
             'transactions/{transactionId}' => function (array $content) {
                 return new Transaction($content);
             },
+            'payouts' => function (array $content) {
```

Comment:
This factory is icorrect and it's not necesary at all. We don;t have `GET` oepartion to `/payouts`